pkg/noderesourcetopology/filter.go

```diff
@@ -81,8 +81,9 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 			// some resources may not expose NUMA affinity (device plugins, extended resources), but all resources
 			// must be reported at node level; thus, if they are not present at node level, we can safely assume
 			// we don't have the resource at all.
-			klog.V(5).InfoS("early verdict", "logKey", logKey, "node", nodeName, "resource", resource, "suitable", "false")
-			return false
+			klog.V(5).InfoS("short circuit", "logKey", logKey, "node", nodeName, "resource", resource)
+			bitmask.Clear()
+			break
```

Comment:
I would prefer to stick to return (the way we currently do) rather than break. Seems more "go" way of doing things to me :)

Reply:
ok, so let's drop this commit.